### ISSUE TITLE
Is it possible to add something like `GroundTruthOracle`

### ISSUE THREAD

**Issue by Yriuns** [sqlancer/sqlancer#820]
for example, we can take `MySQL`'s output as a ground-truth, and use this Oracle to test `TiDB`.
Of course it limits `--num-threads` to be `1`, but I thought it will be a powerful oracle.

**Comment by eltfshr** [comment]
Yes, this oracle is called a differential testing oracle. We at @nus-test are actually working with @mrigger on this oracle right now.
